src/gridcoin/staking/status.h

```diff
@@ -7,53 +7,268 @@
 #include "sync.h"
 #include "uint256.h"
 
+#include <atomic>
 #include <string>
-#include <vector>
+
+class CWalletTx;
 
```

Comment:
`class CWallet`
(for those of us using Qt as the IDE, so that we don't see the pesky parse errors.)

Reply:
Good call!